### ISSUE TITLE
Weekly lover of the week / most loved email

### ISSUE THREAD

**Issue by brenns10** [hacsoc/love#10]
While we can't send out a mass email blast (too many people in the DB, and I don't want to deal with people wanting to be unsubscribed, etc), it would be neat to email the lover/lovee of the week and let them know about it. It would be a really simple cron job, once a week on Monday morning.  The biggest hurdle would be putting together the email template, and especially finding / designing appropriate unicorn and care bear images to go in it!

**Comment by mtbentley** [comment]
I was thinking about this this morning, and one solution is to create a list (love@case.edu?) which is opt-in (via a link somewhere in the footer when you send or receive love?), and email that list + the most loved + the biggest lover each week.

**Comment by brenns10** [comment]
yes, I do like the list idea. it also conveniently skates around the "unsubscribe" issue, and limits the number of emails we send (less $$)

**Comment by brenns10** [comment]
Suggestions on improving this email?

![screenshot_20170303_170709](https://cloud.githubusercontent.com/assets/5682515/23570868/3dfdbe5c-0034-11e7-8e99-42db91bc28e4.png)
![screenshot_20170303_170744](https://cloud.githubusercontent.com/assets/5682515/23570869/3dfe9de0-0034-11e7-856f-47b91747f557.png)

Especially:
- should there be some text, explanation, etc?
- should we mention the mailing list it will be sent out to?
- are the unicorn/care bear images too cheesy?
- improve the layout?

**Comment by brenns10** [comment]
ignore the dev tools overlay :/

**Comment by brenns10** [comment]
Also, check out the horrible code I wrote to render these in dev: https://github.com/hacsoc/love/wiki/Misc
